packages/next/LICENSE

```diff
@@ -1,21 +1,21 @@
 MIT License
```

Comment:
We should probably delete all these individual licenses and just have one in the root.